docs/integrations/sources/pinterest-migrations.md

```diff
@@ -0,0 +1,9 @@
+# Pinterest Migration Guide
+
+## Upgrading to 1.0.0
+
+This release updates date-time fields with airbyte_type: timestamp_without_timezone for streams BoardPins, BoardSectionPins, Boards, Catalogs, CatalogFeeds.
+
+Users should:
```

Comment:
To ensure uninterrupted syncs, users should:
- Refresh the source schema
- Reset affected streams

Reply:
Fixed